.github/workflows/rust.yml

```diff
@@ -73,45 +73,35 @@ jobs:
     name: test wasm32-unknown-unknown
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout sources
-        uses: actions/checkout@v4
-
-      - name: Install stable toolchain
-        uses: dtolnay/rust-toolchain@stable
-
-      - name: Add wasm target
-        run: rustup target add wasm32-unknown-unknown
-
-      - name: Install nodejs v20
-        uses: actions/setup-node@v4
+      - uses: actions/checkout@v4
```

Comment:
What's the rationale for removing all the step names here? Would prefer to split these cosmetic/superficial changes into a separate commit.

Reply:
1. I introduced the step names in the old PR but don't really think they're helping much.
2. All other jobs don't use step names. This is more consistent.